### PR TITLE
@ashfurrow => Add a validation hook for OSS keys

### PR DESCRIPTION
**Pull request by orta**
Checks if keys are not wrong.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Cool! Do we want to do this on before_all, or just before a deploy?

**Comment by orta** [comment]
ah yeah, you use it for more, I'll leave that to you :P

**Comment by ashfurrow** [comment]
Classic Orta.

**Comment by orta** [comment]
https://en.wikipedia.org/wiki/Delegation_pattern

File: fastlane/Fastfile [hunk 1]
```diff
@@ -108,3 +108,28 @@ desc "Updates the storyboard identifier Swift values."
...
+  # Ensure no deploys with OSS keys
+  require 'cocoapods-core'
+  puts "Validating CocoaPods Keys"
+  Dir.chdir ".." do
+    podfile = Pod::Podfile.from_file "Podfile"
+    target = podfile.plugins["cocoapods-keys"]["target"] || ""
+    podfile.plugins["cocoapods-keys"]["keys"].each do |key|
+      puts " - #{key}"
+
+        value = `bundle exec pod keys get #{key} #{target}`
```
Comment (KrauseFx) [inline fastlane/Fastfile]: Uhh, wrong tabs/spaces here..

**Pull request merged**